### DISCUSSION TITLE
Partioned Graph

### DISCUSSION THREAD

**Discussion by lucafrosini** [ArcadeData/arcadedb#361] [Q&A]
I'm evaluating a migration from OrientDB to ArcadeDB.
Does ArcadeDB support partitioned graphs using security at the element (i.e. vertex, edge) level, or any other kind of partitioning mechanism?
I would avoid adding a specific field to use as a filter to identify the element partition(s).

**Comment by lvca** [comment]
Hi Luca,

Could you please provide an example of the partitioning you're looking for?

**Comment by lucafrosini** [comment]
I mean the graph partitioning made using Record Level Security (ORestricted) in OrientDB.

I have a similar scenario (Luca/Steve graph) of the example provided at the following link:
http://orientdb.org/docs/2.0/orientdb.wiki/Partitioned-Graphs.html

**Reply by lvca** [reply]
Record Level security is not available for ArcadeDB but there is a company that wants to sponsor this feature. If they confirm the sponsorship I'll update this discussion with the roadmap.

For now, the only way is to use a property in the vertex/edge.

**Reply by lucafrosini** [reply]
Thanks a lot. I'll follow the project with interest.
